neoexchange/photometrics/gf_movie.py

```diff
@@ -38,7 +38,7 @@
 from django.core.files.storage import default_storage
 
 from photometrics.external_codes import unpack_tarball
-from core.models import Frame, CatalogSources
+from core.models import Frame, CatalogSources, Block
```

Comment:
Can you put `Block` first so that it's in order of increasing specificity ? </petty pedant>